components/analytics/bot-error-summary.tsx

```
@@ -0,0 +1,320 @@
+                case "priority": return uniquePriorities
+                case "platform": return uniquePlatforms
+                default: return []
+            }
+        }
+
+        // Get dimension value for a bot
+        const getDimensionValue = (bot: FormattedBotData, dimension: CrossTabDimension): string => {
+            switch (dimension) {
+                case "category": return bot.status.category || "unknown"
+                case "priority":
+                    const errorType = errorTypes.find(type => type.type === bot.status.value)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)